Hunted Mobile/Hunted Mobile/Hunted Mobile/ViewModel/MapViewModel.cs

```diff
@@ -126,17 +134,6 @@ public class MapViewModel : BaseViewModel {
             this.borderMarkerRepository = borderMarkerRepository;
         }
 
-        private void HandlePinClicked(object sender, PinClickedEventArgs args) {
```

Comment:
Hmmm, wacht. Moet deze functie bovenaan of onderaan? Misschien dat ik dit verkeerd heb gedaan bij de merge-conflicts

Reply:
Onder de constructor en boven de normale methodes. Dus deze en de ICommand methodes zouden wat omhoog moeten.